includes/class-wp-security-bp-database.php

```diff
@@ -133,12 +133,20 @@ public function check_name() {
 		$check = ! in_array( $this->db_name, $this->db_names_blacklist );
 
 		if ( $check ) {
-			$message = 'Your database name ( ' . $this->db_name . ' ) is fine';
-			$this->response->pass( $short_desc, $message );
+			$message = sprintf(
+				/* translators: %s: Name of the database */
+				__( 'Your database name (%s) is fine.', 'wp-security-bp' ),
+				$this->db_name
+			);
+			$this->response->pass( $message, $short_desc );

```

Comment:
OK, I give up. You'll explain me some day why we can't do this in JSON class... because it's so helpful just doing it once.
I'm thinking in something like:
```
$format = 'hi %s, como estás %s?';
$args   = [ 'Calimero', 'amigo' ];

vprintf( $format, $args ); //output: hi Calimero, como estás amigo?
```

so, this means:
```
if ( $check ) {
    $message = 'Your database name %s is fine';
    $args    = array( $this->db_mame );
    $this->response->pass( $message, $short_desc );
} else {
...
```

Reply:
if you agree, I'l commit to this branch before merge. Updating every fail(), pass() ok?
but need to know now, XD

Reply:
I’m afraid this is not gonna work. But you are welcome to test it yourself, just make the changes, install wp-cli and make the .pot file. 😉